submit_and_compare/submit_and_compare.py

```diff
@@ -183,26 +191,28 @@ def studio_view(self, context=None):
             'our_answer_label': self.our_answer_label,
             'submit_button_label': self.submit_button_label,
         }
-        html = self.render_template(
+        html = SubmitAndCompareXBlock.render_template(
```

Comment:
Per elsewhere, if we just make this a top-level module function, all of these calls can be simplified to remove the classname; it's unnecessarily verbose to keep writing `SubmitAndCompareXBlock` when we're already within the SubmitAndCompare XBlock.